### DISCUSSION TITLE
DateInput / DatePickerInput etc. Is there a way to set its popover direction, or to fix its height ?

### DISCUSSION THREAD

**Discussion by keyboqrd** [mantinedev/mantine#5492] [Q&A] [answered]
I recorded two use cases to demonstrate why this could be helpful.

https://github.com/mantinedev/mantine/assets/20307976/6c79a5b2-8440-431e-8de2-1590cdc631af

**Comment by rtivital** [comment, marked as answer]
You can pass props down to the Popover component with `propoverProps`

**Reply by keyboqrd** [reply]
thank you

**Comment by elibroftw** [comment]
This is what worked for me. I suggest putting this in the default Props provider.
```jsx
<DateInput popoverProps={{ styles: { dropdown: { height: '320px' } } }}
```
![image](https://github.com/mantinedev/mantine/assets/21298211/faaa4ee7-dcf0-4c66-b592-d7fb8264371d)
